src/node-opendds.cpp

```diff
@@ -432,15 +422,41 @@ namespace {
       fci.GetReturnValue().SetUndefined();
       return;
     }
+
     void* const dw_i = Nan::GetInternalFieldPointer(fci.This(), 0);
-    const void* const tc_i = Nan::GetInternalFieldPointer(fci.This(), 1);
     DDS::DataWriter* const dw = static_cast<DDS::DataWriter*>(dw_i);
-    const OpenDDS::DCPS::V8TypeConverter* const tc = reinterpret_cast<const OpenDDS::DCPS::V8TypeConverter*>(tc_i);
+
+    const void* const ts_i = Nan::GetInternalFieldPointer(fci.This(), 1);
+    const OpenDDS::DCPS::TypeSupport* const ts = reinterpret_cast<const OpenDDS::DCPS::TypeSupport*>(ts_i);
```

Comment:
Does this need to be a reinterpret?  A typed pointer that goes to `void*` and then comes back can use `static_cast`.